ZeldaFullEditor/Gui/DungeonMain.cs

```diff
@@ -6219,5 +6303,237 @@ private void DungeonMain_KeyUp(object sender, KeyEventArgs e)
+                            if (t != null)
+                            {
+                                t.count++;
+                            }
+                        }
+                    }
+                }
+
+                sx++;
+                if (sx >= 8)
+                {
+                    sy++;
+                    sx = 0;
+                }
+            }
+        }
+
+        // exceptions tiles
+        // 0DA4 to 0DDE
+        // 0918 to 191B
+        // 09BF, 09C0
+        // 09E3
+
+        // animation tiles
+        // 0DDF to 0EA3
+        private void setUnusedTiles16ToToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            if (!setUnusedTiles16ToToolStripMenuItem.Checked)
+            {
+                overworldEditor.showUnusedTile16 = false;
+
+                return;
+            }
+
+            overworldEditor.showUnusedTile16 = true;
+
+            Dictionary<ushort, ushort> alltilesIndexed = new Dictionary<ushort, ushort>();
+            int sx = 0;
+            int sy = 0;
+
+            for (ushort i = 0; i < 4096; i++)
+            {
+                alltilesIndexed.Add(i, 0);
+            }
+
+            for (int i = 0; i < 64; i++)
+            {
+                for (int y = 0; y < 32; y += 1)
+                {
+                    for (int x = 0; x < 32; x += 1)
+                    {
+                        ushort LWTile = overworldEditor.overworld.AllMapTile32LW[x + (sx * 32), y + (sy * 32)];
+                        alltilesIndexed[LWTile]++;
+                        ushort DWTile = overworldEditor.overworld.AllMapTile32DW[x + (sx * 32), y + (sy * 32)];
+                        alltilesIndexed[DWTile]++;
+
+                        if (i < 32)
+                        {
+                            alltilesIndexed[overworldEditor.overworld.AllMapTile32SP[x + (sx * 32), y + (sy * 32)]]++;
+                        }
+                    }
+                }
+
+                foreach (TilePos t in overworldEditor.overworld.AllOverlays[i].TileDataList)
+                {
+                    alltilesIndexed[t.tileId]++;
+                }
+
+                foreach (TilePos t in overworldEditor.overworld.AllOverlays[i + 64].TileDataList)
+                {
+                    alltilesIndexed[t.tileId]++;
+                }
+
+                sx++;
+                if (sx >= 8)
+                {
+                    sy++;
+                    sx = 0;
+                }
+            }
+            // exceptions tiles
+            // 0DA4 to 0DDE
+            // 0918 to 091B
+            // 09BF, 09C0
+            // 09E3
+
+            // animation tiles
+            // 0DDF to 0EA3
+
+            List<ushort> exceptionsTiles = new List<ushort>();
+            for(int i = 0xDA4; i <= 0x0DDE; i++)
+            {
+                exceptionsTiles.Add((ushort)i);
+            }
+
+            exceptionsTiles.Add(0x918);
```

Comment:
constants to label all these exception tiles

Reply:
idk what this is about, this is for zarby lol